lib/process-services-cloud/src/lib/form/components/form-cloud.component.ts

```diff
@@ -376,10 +430,18 @@ export class FormCloudComponent extends FormBaseComponent implements OnChanges,
         return !args.defaultPrevented;
     }
 
-    protected storeFormAsMetadata() {}
+    protected storeFormAsMetadata() { }
```

Comment:
this might need to be skipped, empty method

Reply:
We need to provide an implementation of this `storeFormAsMetadata` method that is inherited from the `abstract` parent class. I just formatted this file, I did not change its implementation